data_subscriptions/worker/latest_ckan_activity.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, url=None, start_time=None, proposed_limit=1_000):
         self.current_offset = 0
 
     def __call__(self):
-        api = RemoteCKAN(self.url)
+        api = RemoteCKAN(self.url, user_agent='ckanapiexample/1.0 (+http://example.com/my/website)')
```

Comment:
@sagargg why not to use actual domain?

Reply:
@anuveyatsu  not sure but found mostly user-age in this pattern. 
`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.97 Safari/537.36`

Reply:
@sagargg I'm not sure if it is needed at all, can we we just not do it?